### PR TITLE
Added models and repos for Connected Devices

### PR DESCRIPTION
**Pull request by niksv**
NOTE: migration scrips were added

Part of #2582

### CODE REVIEW

File: services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    @Setter(AccessLevel.NONE)
+    @Getter(AccessLevel.NONE)
+    private Long entityId;
+
+    @NonNull
+    @Index(unique = true)
+    @Property(name = "flow_id")
+    private String flowId;
+
+    @Property(name = "src_lldp_meter_id")
```
Comment (sergii-iakovenko) [inline services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java]: This is already LLDP-related entity. Can't we remove "lldp" from the property names? Like src_meter_id & dst_meter_id

File: services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    @Convert(graphPropertyType = Long.class)
+    private MeterId srcLldpMeterId;
+
+    @Property(name = "dst_lldp_meter_id")
+    @Convert(graphPropertyType = Long.class)
+    private MeterId dstLldpMeterId;
```
Comment (sergii-iakovenko) [inline services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java]: Shouldn't XXXmeterId fields be non-nullable?

File: services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java (hunk 1 above)
Comment (niksv) [reply services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java]: fixed

File: services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java (hunk 2 above)
Comment (niksv) [reply services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/LldpResources.java]: we can enable LLDP only for Flow source. In this case dstMeter will be null

File: services/neo4j/migrations/1.9-connected-devices/1-update-constraints-changelog.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    <!-- and/or define directly changesets-->
+    <changeset id="1.9-add-index-and-constraint-connected-device" author="Sergey Nikitin">
+        <query><![CDATA[
+            CREATE CONSTRAINT ON (connected_device:connected_device) ASSERT connected_device.entityId IS UNIQUE
+        ]]></query>
+        <query><![CDATA[
+            CREATE CONSTRAINT ON (connected_device:connected_device) ASSERT connected_device.unique_index IS UNIQUE
+        ]]></query>
+        <query><![CDATA[
+            CREATE INDEX ON :connected_device (flow_id);
```
[the remark this answers is not in the document]
Comment (niksv) [reply services/neo4j/migrations/1.9-connected-devices/1-update-constraints-changelog.xml]: fixed

**Pull request merged**